site/content/docs/latest/howto/private-app-repository.md

```diff
@@ -1,220 +1,248 @@
+- `Scope`: may be either **global** (cluster﹣wide) or **namespace﹣scoped**, in which case only users with access to that namespace will have access to the repository.
+- `Package storage type`: Helm repository or OCI registry.
 
-To use Harbor with Kubeapps, first deploy the [Bitnami Harbor Helm Chart](https://github.com/bitnami/charts/tree/master/bitnami/harbor) from the `bitnami` repository (alternatively you can deploy Harbor using [Harbor offline installer](https://goharbor.io/docs/2.5.0/install-config/download-installer/)):
+For private repositories, the Authentication method must be provided. In this case select:
 
-![Harbor Chart](../img/harbor-chart.png)
+- `Basic Auth` (for `Repository Authorization` field under `Authorization` tab) and specify the username and password for Harbor.
 
-In the deployment form we should change the parameter below:
+![Harbor private registry](../img/harbor-private-registry.png)
 
-- `service.tls.enabled`: We should set this value to `false` so we don't need to configure the TLS settings. Alternatively, you can provide valid TSL certificates (check [Bitnami Harbor Helm Chart documentation](https://github.com/bitnami/charts/tree/master/bitnami/harbor#parameters) for more information).
+Once configured, you will be able to browse and deploy your own applications from the private Harbor registry by using Kubeapps (and see the chart you uploaded in the previous step).
 
-![Harbor Deploy Form](../img/harbor-deploy-form.png)
+## OCI Registry
 
-Deploy the chart and wait for it to be ready.
+It is also possible to use an [OCI compliant registry](https://github.com/opencontainers/distribution-spec) to store packages.
 
-![Harbor Chart Ready](../img/harbor-ready.png)
+Cloud Providers like [Amazon ECR](https://aws.amazon.com/blogs/containers/oci-artifact-support-in-amazon-ecr/), [Google Artifact Registry](https://cloud.google.com/artifact-registry/docs/supported-formats) or [Azure ACR](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-image-formats) have also added support for OCI artifacts.
 
-**Note**: By default, Harbor deploys ChartMuseum to store charts. If you deactivate it, you can still use the Harbor OCI registry to upload charts. Check out the [OCI](#oci-registry) section for more information.
+You can add an OCI registry like any other repository. For example, for a [Harbor registry](https://goharbor.io/docs/2.5.0/working-with-projects/working-with-images/managing-helm-charts/#manage-helm-charts-with-the-oci-compatible-registry-of-harbor):
 
-### Harbor: Upload a Chart
+![OCI registry](../img/oci-registry.png)
 
-First create a Helm chart package:
+**NOTE:** There is one caveat though. It's necessary to specify the list of applications (repositories) that the registry contains. This is because the OCI specification doesn't have an endpoint to discover artifacts (unlike the index.yaml file of a Helm repository).
```

Comment:
```suggestion
**NOTE:** There is one caveat though. It is necessary to specify the list of applications (repositories) that the registry contains. This is because the OCI specification doesn't have an endpoint to discover artifacts (unlike the index.yaml file of a Helm repository).
```